src/salesforce/salesforce-adapter.ts

```diff
@@ -44,6 +47,15 @@ export class SalesforceAdapter
     articleId: string | null,
     url: string,
   ): Promise<Image | null> {
+    if (!SalesforceAdapter.ATTACHMENT_RELATIVE_PATH_REGEX.test(url)) {
+      if (url.startsWith('/')) {
+        return Promise.resolve(null);
```

Comment:
Personally I prefer returning null and making the method async which will wrap it into a Promise.
This Promise.resolve call is just noise.

Reply:
Reworked and not matching relative urls and absolute urls are handled the same way now